### PR TITLE
[All hosts](SSO) Refresh app registration steps

### PR DESCRIPTION
**Pull request by davidchesnut**
Updated app registration steps and added additional screen shots for clarity.

### CODE REVIEW

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 1]
```diff
@@ -1,64 +1,126 @@
...
+
+You need to create an app registration in Azure that represents your web server. This enables authentication support so that proper access tokens can be issued to the client code in JavaScript. This registration supports both SSO in the client, and fallback authentication using the Microsoft Authentication Library (MSAL).
 
-Registering your application (the add-in) establishes a trust relationship between your add-in and the Microsoft identity platform. The trust is unidirectional: your add-in trusts the Microsoft identity platform, and not the other way around.
 
 1. Sign in to the [Azure portal](https://portal.azure.com/) with the ***admin*** credentials to your Microsoft 365 tenancy. For example, **MyName@contoso.onmicrosoft.com**.
-1. Under **Manage**, select **App registrations** > **New registration**. On the **Register an application** page, set the values as follows.
+1. Select the **App registrations** icon. If you don't see the icon, search for "app registration" in the search bar.
+
+    :::image type="content" source="../images/azure-portal-select-app-registration.png" alt-text="The Azure portal home page.":::
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:9]: azure-portal-add-application
I cut out the extra white space. Please use this version instead! There's a couple other "spacey" images, but this one was the most egregious.

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 2]
```diff
@@ -67,28 +129,69 @@ Sometimes called an _application password_, a client secret is a string value yo
     > - `93d53678-613d-4013-afc1-62e9e444a0a5` (Office on the web)
     > - `bc59ab01-8403-45c6-8796-ac3ef710b3e3` (Outlook on the web)
 
-1. Select **Add a client application**. In the panel that opens, set the **Client ID** to the respective GUID and check the box for `api://<fully-qualified-domain-name>/<application-id>/access_as_user`.
+1. Select the checkbox for `api://localhost:44355/<app-id>/access_as_user` in **Authorized scopes**.
 
 1. Select **Add application**.
 
+    :::image type="content" source="../images/azure-portal-add-application.png" alt-text="The Add a client application pane.":::
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:136]: azure-portal-add-application


I cut out the extra white space. Please use this version instead!

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 3]
```diff
@@ -67,28 +129,69 @@ Sometimes called an _application password_, a client secret is a string value yo
...
+
+    * Files.Read
     * openid
     * profile
 
     > [!NOTE]
     > The `User.Read` permission may already be listed by default. It's a good practice to only request permissions that are needed, so we recommend that you uncheck the box for this permission if your add-in does not actually need it.
 
-1. Select the check box for each permission as it appears (note that the permissions will not remain visible in the list as you select each one). After selecting the permissions that your add-in needs, select the **Add permissions** button.
+1. Select the checkbox for each permission as it appears (note that the permissions will not remain visible in the list as you select each one). After selecting the permissions that your add-in needs, select the **Add permissions** button.
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:169]:
```suggestion
1. Select the checkbox for each permission as it appears. Note that the permissions will not remain visible in the list as you select each one. After selecting the permissions that your add-in needs, select the **Add permissions** button.
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 4]
```diff
@@ -67,28 +129,69 @@ Sometimes called an _application password_, a client secret is a string value yo
...
+
+    :::image type="content" source="../images/azure-portal-request-api-permissions-add-permissions.png" alt-text="The Request API permissions pane with some permissions selected.":::
+
+## Configure access token version
+
+You must define the access token version that is acceptable for your app. This configuration is made in the Azure Active Directory application manifest.
+
+### To define the access token version
+
+the access token version can change if you chose an account type other than **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**. Use the following steps to ensure the access token version is correct for Office SSO usage.
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:179]:
```suggestion
The access token version can change if you chose an account type other than **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**. Use the following steps to ensure the access token version is correct for Office SSO usage.
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 5]
```diff
@@ -67,28 +129,69 @@ Sometimes called an _application password_, a client secret is a string value yo
...
-1. Select the check box for each permission as it appears (note that the permissions will not remain visible in the list as you select each one). After selecting the permissions that your add-in needs, select the **Add permissions** button.
+1. Select the checkbox for each permission as it appears (note that the permissions will not remain visible in the list as you select each one). After selecting the permissions that your add-in needs, select the **Add permissions** button.
+
+    :::image type="content" source="../images/azure-portal-request-api-permissions-add-permissions.png" alt-text="The Request API permissions pane with some permissions selected.":::
+
+## Configure access token version
+
+You must define the access token version that is acceptable for your app. This configuration is made in the Azure Active Directory application manifest.
+
+### To define the access token version
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:177]:
```suggestion
### Define the access token version
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 6]
```diff
@@ -1,64 +1,126 @@
-## Create an app registration
+## Register the add-in with Microsoft identity platform
+
+You need to create an app registration in Azure that represents your web server. This enables authentication support so that proper access tokens can be issued to the client code in JavaScript. This registration supports both SSO in the client, and fallback authentication using the Microsoft Authentication Library (MSAL).
 
-Registering your application (the add-in) establishes a trust relationship between your add-in and the Microsoft identity platform. The trust is unidirectional: your add-in trusts the Microsoft identity platform, and not the other way around.
 
 1. Sign in to the [Azure portal](https://portal.azure.com/) with the ***admin*** credentials to your Microsoft 365 tenancy. For example, **MyName@contoso.onmicrosoft.com**.
-1. Under **Manage**, select **App registrations** > **New registration**. On the **Register an application** page, set the values as follows.
+1. Select the **App registrations** icon. If you don't see the icon, search for "app registration" in the search bar.
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:7]:
```suggestion
1. Select **App registrations**. If you don't see the icon, search for "app registration" in the search bar.
```
Per MSFT style, refer to UI elements by their label and not the element. Unless it's totally confusing not to do so.
https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33421

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 7]
```diff
@@ -1,64 +1,126 @@
...
+
+    :::image type="content" source="../images/azure-portal-select-app-registration.png" alt-text="The Azure portal home page.":::
+
+    The **App registrations** page appears.
+
+1. Select the **New registration** icon.
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:13]:
```suggestion
1. Select **New registration**.
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 8]
```diff
@@ -1,64 +1,126 @@
...
+    The section for setting the application ID URI appears with a generated Application ID URI in the form `api://<app-id>`.
+
+1. Update the application ID URI to `api://localhost:44355/<app-id>`.
+
+    :::image type="content" source="../images/azure-portal-app-id-uri-details.png" alt-text="Edit the App ID URI pane with localhost port set to 44355.":::
+
+    * The **Application ID URI** is pre-filled with app ID (GUID) in the format `api://<app-id>`.
+    * The application ID URI format should be: `api://<fully-qualified-domain-name>/<app-id>`
+    * Insert the `fully-qualified-domain-name` between `api://` and `<app-id>` (which is a GUID). For example, `api://contoso.com/<app-id>`.
+    * If you are using localhost, then the format should be `api://localhost:<port>/<app-id>`. For example, `api://localhost:44355/c6c1f32b-5e55-4997-881a-753cc1d563b7`.
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:78]:
```suggestion
    * If you're using localhost, then the format should be `api://localhost:<port>/<app-id>`. For example, `api://localhost:44355/c6c1f32b-5e55-4997-881a-753cc1d563b7`.
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 9]
```diff
@@ -1,64 +1,126 @@
...
-1. Select the **Add a scope** button. In the panel that opens, enter `access_as_user` as the **Scope name**.
+1. Select the **Add a scope** button.
+
+    :::image type="content" source="../images/azure-portal-add-a-scope.png" alt-text="Select Add a scope button.":::
 
-1. Set **Who can consent?** to **Admins and users**.
+    The **Add a scope** pane opens.
 
-1. Fill in the fields for configuring the admin and user consent prompts with values that are appropriate for the `access_as_user` scope which enables the Office client application to use your add-in's web APIs with the same rights as the current user. Suggestions:
+1. Specify the scope's attributes in the **Add a scope** pane.
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:93]:
```suggestion
1. In the **Add a scope** pane, specify the scope's attributes .
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 10]
```diff
@@ -1,64 +1,126 @@
...
-    * **Admin consent description:** Enable Office to call the add-in's web APIs with the same rights as the current user.
-    * **User consent display name:** Office can act as you.
-    * **User consent description:** Enable Office to call the add-in's web APIs with the same rights that you have.
+    :::image type="content" source="../images/azure-portal-add-a-scope-details.png" alt-text="Add a scope pane with example values.":::
 
-1. Ensure that **State** is set to **Enabled**.
+    | Field | Description | Values |
+    |-------|-------------|---------|
+    | **Scope name** | The name of your scope. A common scope naming convention is `resource.operation.constraint`. | For SSO this must be set to `access_as_user`. |
+    | **Who can consent** | Whether this scope can be consented to by users or if admin consent is required. Select **Admins only** for higher-privileged permissions. | For learning SSO and samples, we recommend you set this to **Admins and users**. |
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:100]:
```suggestion
    | **Who can consent** |  Determines if admin consent is required or if users can consent without an admin approval. | For learning SSO and samples, we recommend you set this to **Admins and users**. <br><br>Select **Admins only** for higher-privileged permissions.|
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 11]
```diff
@@ -1,64 +1,126 @@
...
+    | **Admin consent display name** | A short description of the scope's purpose that only admins will see. | `Read-only access to user files and profiles.` |
+    | **Admin consent description** | A more detailed description of the permission granted by the scope that only admins will see. | `Allow Office to have read-only access to all user files and profiles. Office can call the app's web APIs as the current user.` |
```
Comment (lindalu-MSFT) [inline docs/includes/register-sso-add-in-aad-v2-include.md:102]:
```suggestion
    | **Admin consent description** | A more detailed description of the permission granted by the scope that only admins see. | `Allow Office to have read-only access to all user files and profiles. Office can call the app's web APIs as the current user.` |
```

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]
A few things, and let me know if you need any help resizing screenshots. I did revise one with lots of whitespace and dropped it in one of my comments.

File: docs/includes/register-sso-add-in-aad-v2-include.md (hunk 2 above)
Comment (davidchesnut) [reply docs/includes/register-sso-add-in-aad-v2-include.md:136]: Done.

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**